docs/3.api/3.utils/$fetch.md

```diff
@@ -10,9 +10,22 @@ Nuxt uses [ofetch](https://github.com/unjs/ofetch) to expose globally the `$fetc
 ::ReadMore{link="/docs/getting-started/data-fetching"}
 ::
 
-During server-side rendering, calling `$fetch` to fetch your internal [API routes](/docs/guide/directory-structure/server) will directly call the relevant function (emulating the request), **saving an additional API call**.
+During server-side rendering, calling `$fetch` **on the server side** to fetch your internal [API routes](/docs/guide/directory-structure/server) will directly call the relevant function (emulating the request), **saving an additional API call**. However, when using server-side rendering, calling `$fetch` *on the client side* without wrapping it with `useAsyncData` causes fetching the data twice initially on the server, then again on the client, because `$fetch` does not transfer state from the server to the client. Thus, the fetch will be executed on both sides because the client has to get the data again. Use [`useFetch`](https://nuxt.com/docs/api/composables/use-fetch) or [`useAsyncData`](https://nuxt.com/docs/api/composables/use-async-data) with `$fetch` on the client side to prevent double data fetching.
```

Comment:
I think we need to reduce the **on the client side** or explain the hydration phase (which is when Vue is mounting the app on cient-side after SSR).

Because **on the client side** can be confusing, what if you come from page A to page B, page B is using `$fetch` but only one call will be made. If he refreshes page B, then two call will be made (server + client hydration).